Document GitOps RN-1.8.3

Srivaralakshmi opened the pull request:
**NOTE**: From GitOps 1.7.0 onwards, we are documenting security issues in a new advisory format as part of errata updates.

**Aligned team**: Dev Tools

**Purpose**: To resolve the following issues:
(https://issues.redhat.com/browse/RHDEVDOCS-5216)


**OCP version this PR applies to**: enterprise-`4.10` and later

**Link to docs preview**: 
- [Release notes for Red Hat OpenShift GitOps 1.8.3]()


**SME review**: @iam-veeramalla, @jaideepr97 
**QE review**: @varshab1210
**Peer-review**:

Srivaralakshmi commented:
@iam-veeramalla, @jaideepr97, and @varshab1210: Please review this PR and provide feedback.

Please let me know if there are any more issues to be covered in this Release Notes for the 1.8.3 release. Thank you!

Srivaralakshmi commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,32 @@
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```
@jaideepr97 Addressed in the revised PR. PTAL.

Srivaralakshmi commented:
@iam-veeramalla, @jaideepr97, @harrietgrace, and @varshab1210: Here is the revised PR. 

Varsha, PTAL at the Note on `stable` channel.
Jaideep, I have addressed your suggestion PTAL.

- If there is anything more to be covered in RN 1.8.3, please let me know right away.
- If the content looks good, please provide approval. I will send the PR to peer review and keep it ready for the merge tomorrow.

varshab1210 commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,37 @@
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
```
@Srivaralakshmi I am not able to open the link from preview page

Srivaralakshmi replied:
FYI, I am providing the following links:

- https://access.redhat.com/errata/RHBA-2023:112946
- https://access.redhat.com/errata/RHBA-2023:112945

These links go to the Customer Portal, not to the Errata Tool. The Errata Tool is internal only.
Users wouldn't even be able to access it.
Now you are seeing **404 Page not found** as they are not released yet. See their status. It says NEW FILES. They will only become visible on the CP when released, i.e. when their status is SHIPPED LIVE.
When released, the errata will have URLs like this: https://access.redhat.com/errata/RHBA-2023:112945.

jaideepr97 commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,37 @@
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` ...
```
```suggestion
* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not correctly propagated to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled, and HPA parameters are correctly updated.  
link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

Srivaralakshmi replied:
@jaideepr97 Addressed in the revised PR. PTAL.

Srivaralakshmi commented:
@iam-veeramalla, @jaideepr97, @harrietgrace, and @varshab1210: Here is the revised PR.

Varsha, as discussed, I have moved the admonition just before the matrix table - PTAL.
Jaideep, I have addressed your suggestion PTAL.

- If there is anything more to be covered in RN 1.8.3, please let me know right away.
- If the content looks good, please provide approval. I will send the PR to peer review and keep it ready for the merge tomorrow.

jaideepr97 reviewed (approved):
lgtm, thanks @Srivaralakshmi

Srivaralakshmi commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,32 @@
+
+The list of security fixes that are included in this release is documented in the following advisories:
+
+* link:https://access.redhat.com/errata/RHEA-2023:112944[RHEA-2023:112944]
+* link:https://access.redhat.com/errata/RHBA-2023:112945[RHBA-2023:112945]
```
@reginapizza  and @iam-veeramalla: The Errata links are not accessible from the preview page. I think they are still not fixed? don't know if this is some permission issue?
https://access.redhat.com/errata/RHEA-2023:112944

No idea why. I need the errata links for them.

Any help here is much appreciated. TIA : )

varshab1210 replied:
@Srivaralakshmi LGTM from my side once we get appropriate links

Srivaralakshmi replied:
FYI, I am providing the following links:

- https://access.redhat.com/errata/RHBA-2023:112946
- https://access.redhat.com/errata/RHBA-2023:112945

These links go to the Customer Portal, not to the Errata Tool. The Errata Tool is internal only.
Users wouldn't even be able to access it.
Now you are seeing **404 Page not found** as they are not released yet. See their status. It says NEW FILES. They will only become visible on the CP when released, i.e. when their status is SHIPPED LIVE.
When released, the errata will have URLs like this: https://access.redhat.com/errata/RHBA-2023:112945.

Srivaralakshmi commented:
@varshab1210 Please approve the PR and provide `/lgtm`.

rolfedh commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,32 @@
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica setting in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` ...
```
Possibly typo:
```suggestion
* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica set in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale` is disabled and the HPA parameters are updated correctly. link:https://issues.redhat.com/browse/GITOPS-2629[GITOPS-2629]
```

Srivaralakshmi replied:
@rolfedh yes, changed it to settings. Thanks!

rolfedh commented on modules/go-compatibility-and-support-matrix.adoc:
```diff
@@ -14,11 +14,16 @@ In the table, features are marked with the following statuses:
 
+[IMPORTANT]
+====
+Before upgrading to {product-title} 4.13, move out of the `stable` channel.
```
The expression "move out" doesn't seem to fit here and isn't used elsewhere in the openshift-docs repo. Is there another more common way of expressing the same idea?

Srivaralakshmi replied:
@rolfedh Addressed. Thank you!

rolfedh reviewed (commented):
Hi @Srivaralakshmi. I've offered a suggestion and a question. HTH! Nice work!

varshab1210 commented:
LGTM
Thanks

iam-veeramalla commented:
/lgtm

rolfedh reviewed (approved):
Looks good to me. Nice work.

Srivaralakshmi commented:
@varshab1210 Can you please approve the PR?

varshab1210 commented:
/lgtm
Thanks

iam-veeramalla commented on modules/gitops-release-notes-1-8-3.adoc:
```diff
@@ -0,0 +1,32 @@
+[id="fixed-issues-1-8-3_{context}"]
+== Fixed issues
+
+* Before this update, when `Autoscale` was enabled and the horizontal pod autoscaler (HPA) controller tried to edit the replica settings in server deployment, the Operator overwrote it. In addition, any changes specified to the autoscaler parameters were not propagated correctly to the HPA on the cluster. This update fixes the issue. Now the Operator reconciles on replica drift only if `Autoscale ...
```
is overwrote a correct word to use ?

Srivaralakshmi replied:
yes.

gabriel-rh commented:
/cherry-pick enterprise-4.10

gabriel-rh commented:
/cherry-pick enterprise-4.11

gabriel-rh commented:
/cherry-pick enterprise-4.12

gabriel-rh commented:
/cherry-pick enterprise-4.13

gabriel-rh commented:
/cherry-pick enterprise-4.14
